src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/ContentHelper.CoreClr.cs

```diff
@@ -14,13 +14,26 @@ internal static partial class ContentHelper
     {
         internal static Encoding GetEncoding(HttpResponseMessage response)
         {
-            string charSet = response.Content.Headers.ContentType.CharSet;
+                
+            //ContentType may not exist in response header.
+            string charSet = null;
+            if ( response.Content.Headers.ContentType != null)
+            { 
+                charSet = response.Content.Headers.ContentType.CharSet;
+            }
```

Comment:
C# 6 has a nice sugar for this, [Null-Conditional Operator](https://msdn.microsoft.com/en-us/magazine/dn802602.aspx)

```csharp
string charSet = response.Content.Headers.ContentType?.CharSet;
```